src/vcpkg.cpp

```diff
@@ -63,6 +63,9 @@ static void invalid_command(const std::string& cmd)
 
 static void inner(vcpkg::Filesystem& fs, const VcpkgCmdArguments& args)
 {
+    // track version on each invocation
+    LockGuardPtr<Metrics>(g_metrics)->track_property("vcpkg_version", Commands::Version::version.to_string());
```

Comment:
```suggestion
    LockGuardPtr<Metrics>(g_metrics)->track_property("vcpkg-version", Commands::Version::version.to_string());
```

I think the other ones use dashes?

Reply:
I've become a proponent of  using `_` since you can query like 

```
where Properties.vcpkg_version == '2022-01-01'
``` 

as opposed to

```
where Properties['vcpkg-version'] == '2022-01-01'
```

Reply:
OK, we should make sure to follow the same convention for anything new.